packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -339,6 +340,68 @@ export const PSM_GOV_MANIFEST = {
   },
 };
 
+/** @type { <X, Y>(xs: X[], ys: Y[]) => [X, Y][]} */
+const zip = (xs, ys) => xs.map((x, i) => [x, ys[i]]);
+
+/**
+ * @param {import('./econ-behaviors').EconomyBootstrapPowers} powers
+ * @param {{ options: { voterAddresses: Record<string, string> }}} param1
+ */
+export const invitePSMCommitteeMembers = async (
+  {
+    consume: { zoe, namesByAddressAdmin, economicCommitteeCreatorFacet },
+    installation: {
+      consume: { binaryVoteCounter: counterP, psmCharter: psmCharterP },
+    },
+  },
+  { options: { voterAddresses } },
+) => {
+  /** @type {[Installation, Installation]} */
+  const [charterInstall, counterInstall] = await Promise.all([
+    psmCharterP,
+    counterP,
+  ]);
+  const terms = await deeplyFulfilledObject(
+    harden({
+      binaryVoteCounterInstallation: counterInstall,
+    }),
+  );
+
+  const { creatorFacet } = E.get(
+    E(zoe).startInstance(charterInstall, undefined, terms),
+  );
+
+  const invitations = await E(
+    economicCommitteeCreatorFacet,
+  ).getVoterInvitations();
+  assert.equal(invitations.length, values(voterAddresses).length);
+
+  /**
+   * @param {[string, Promise<Invitation>][]} addrInvitations
+   */
+  const distributeInvitations = async addrInvitations => {
+    await Promise.all(
+      addrInvitations.map(async ([addr, invitationP]) => {
+        const [voterInvitation, charterMemberInvitation] = await Promise.all([
+          invitationP,
+          E(creatorFacet).makeCharterMemberInvitation(),
+        ]);
+        console.log('@@@sending charter, voting invitations to', addr);
```

Comment:
```suggestion
        console.log('sending charter, voting invitations to', addr);
```

That `@@@` thingy is my "I have to run to a meeting now; make sure I don't ship this half-baked code" marker. Let's take it out. Or we can drop the log message altogether, but I think maybe this is worth keeping.

Reply:
done